templates/travis/.travis/before_install.sh.j2

```diff
@@ -123,10 +93,25 @@ if [ -z "$TRAVIS_TAG" ]; then
 fi
 {% endif %}
 
-pip install ansible
+{%- for plugin in additional_plugins %}
+
+git clone --depth=1 https://github.com/pulp/{{ plugin.name }}.git --branch {{ plugin.branch }}
+if [ -n "${{ plugin.name | upper | replace("-", "_") }}_PR_NUMBER" ]; then
+  cd {{ plugin.name }}
+  git fetch --depth=1 origin pull/${{ plugin.name | upper | replace("-", "_") }}_PR_NUMBER/head:${{ plugin.name | upper | replace("-", "_") }}_PR_NUMBER
+  git checkout ${{ plugin.name | upper | replace("-", "_") }}_PR_NUMBER
+  cd ..
+fi
+{%- endfor %}
+
+# Intall requirements for ansible playbooks
+pip install docker netaddr boto3
+
+# Install ansible with the boto3 tags to dict fix
+pip install git+https://github.com/mdellweg/ansible.git@fix_boto3_tags_dict
```

Comment:
Do we have a PR against ansible?

I looked for one.

It also looks like ec2.py is no longer in the repo as of 2.10 devel (moved to a partner collection?)

Reply:
Yeah, i did not find that file either in master. My guess is the humongous community-collection.
But actually i am unsure, whether the root problem here is with ansible, or minio.

Reply:
I see what you mean.

I found the repo for Ansible 2.10 (`git log -- ./deleted/file/path`), the line is not fixed:
https://github.com/ansible-collections/amazon.aws/blob/master/plugins/module_utils/ec2.py#L458

[It is published on Galaxy here](https://galaxy.ansible.com/amazon/aws), per Ansible's plan to separate partner modules from ansible core:

Could you please try reporting it to 1 or the other?

Reply:
Will do. Thank you for finding it.

Reply:
https://github.com/ansible-collections/amazon.aws/pull/37

Reply:
Great. Please do 1 of the following so we don't end up with this forever:
1. Put a comment above it with a link to the PR.
2. Create a task in redmine to make sure the PR gets accepted there (or in minio if they refuse.)

Reply:
Comment in place. I mentioned the move of the code to the collection, because this might be interesting anyway, when using the next version of ansible.